.github/workflows/release.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - master
+      - '[0-9]+.x'
```

Comment:
should these be added to master as well?

Reply:
I think it probably should. I can make a different PR to add it.